### PR TITLE
Porting intro documentation language reworked

### PR DESCRIPTION
**Pull request by comradekingu**
@doniks
Can continue fixing up the others too.

### CODE REVIEW

File: porting/introduction/Preparations.rst [hunk 1]
```diff
@@ -3,43 +3,59 @@
...
 ============
 
-Not all devices can be made to run Ubuntu Touch. Research your target device before you start.
+Not all devices can be made to run Ubuntu Touch.
+Research your specific target device before starting.
 
-To determine your device's specifications, search for it on `GSM Arena <https://www.gsmarena.com>`_ and/or `Device Specification <https://www.devicespecifications.com/en>`_.
-Refer to `Pick an Android target device in the Halium Porting Guide <http://docs.halium.org/en/latest/porting/first-steps.html#pick-an-android-target-device>`_ for further information on requirements and how to check if your device qualifies.
+Find the specifications of your device by searching for it on `GSMArena <https://www.gsmarena.com>`_ and/or `DeviceSpecification <https://www.devicespecifications.com/en>`_.
+`Pick an Android target device<http://docs.halium.org/en/latest/porting/first-steps.html#pick-an-android-target-device>`_ in the Halium Porting Guide helps you find out if it meets the requirements.
```
Comment (doniks) [inline porting/introduction/Preparations.rst:10]: It fails to build here https://ci.ubports.com/blue/organizations/jenkins/docs.ubports.com/detail/PR-524/4/pipeline
Not sure why though :( maybe the deleted space after "device"?

**Review by doniks — CHANGES_REQUESTED** [review]
doesn't built right now. pls check below (haven't reviewed any further yet)

File: porting/introduction/Preparations.rst (hunk 1 above)
Comment (comradekingu) [inline porting/introduction/Preparations.rst:10]:
```suggestion
`Pick an Android target device <https://docs.halium.org/en/latest/porting/first-steps.html#pick-an-android-target-device>`_ in the Halium Porting Guide helps you find out if it meets the requirements.
```

**Comment by doniks** [comment]
it has conflicts now

**Comment by doniks** [comment]
ping?

**Comment by comradekingu** [comment]
@doniks It stopped complaining (?). Just undid the conflict, and changed the line-shifting around.

**Comment by doniks** [comment]
still complaining :thinking: 
image

**Comment by comradekingu** [comment]
@doniks You must have missed the "no complaints" merge window X)
It is friendly now, after sending me for a tribute run.

**Comment by doniks** [comment]
cleaning up old open PR's ... in case you want to refloat this feel free to reopen!

**Pull request closed without merging**